.github/workflows/build-test.yml

```
@@ -61,13 +61,17 @@ jobs:
       with:
         go-version-file: .go-version
 
+    - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # v3.0.0
+      with:
+        version: 8.15.6
```

Could you set the version from the env and add a comment for Renovate like we do for other tools?

```suggestion
        version: ${{ env.PNPM_VERSION }}
```

```yaml
env:
  # renovate: datasource=npm depName=pnpm versioning=npm
  PNPM_VERSION: 8.15.6
```

---

Thanks for catching this, updated other workflow files as well.